riotfile.py

```diff
@@ -1552,7 +1552,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         ),
         Venv(
             name="pytest",
-            command="pytest --no-ddtrace {cmdargs} tests/contrib/pytest/",
+            command="pytest --no-ddtrace --no-cov {cmdargs} tests/contrib/pytest/",
```

Comment:
Is this related to the incompatibility between ITR and pytest coverage?

Reply:
It might be, but I don't know.